src/index.js

```diff
@@ -98,4 +101,17 @@ app.get('/deletePledge/:pledgeId', async ({ params: { pledgeId } }, res) => {
   }
 });
 
+async function findNewNotifications() {
+  // notify for pledges that have expired
+  const expiredPledges = await db.findAllPledgesExpiredToNotify();
+  expiredPledges.map(async p => {
```

Comment:
just a note (in this case it shouldn't be an issue): putting the `async/await` inside the map function doesn't mean you're waiting for it so `findNewNotifications` returns before the map has actually ended.

As I already said, as the interval is set for 1min, it's not an issue in this case